### PR TITLE
build 1.2.0

### PR DESCRIPTION
**Pull request by boldorider4**
* [Upstream repo](https://github.com/lapets/bitlist)
* [Jira ticket](https://anaconda.atlassian.net/browse/PKG-3120)

### CODE REVIEW

**Review by JeanChristopheMorinPerso — APPROVED** [review]

**Review by Arishamays1 — APPROVED** [review]
LGTM

File: recipe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+about:
+  home: https://github.com/lapets/bitlist
+  license: MIT
+  license_file: LICENSE
+  license_family: MIT
+  summary: Pure-Python library for working with bit vectors.
+  description: |
+    This library allows programmers to work with bit vectors using a pure-Python data structure. Its design prioritizes
+    interoperability with built-in Python classes and operators.
+  doc_url: https://github.com/lapets/bitlist/blob/main/README.rst
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml]:
```suggestion
  doc_url: https://bitlist.readthedocs.io/en/1.2.0/
```

Comment (Arishamays1) [reply recipe/meta.yaml]: Thanks JC!

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
LGTM

**Pull request merged**